### ISSUE TITLE
Request: Auto refresh...

### ISSUE THREAD

**Issue by MrMxyzptlk** [causefx/Organizr#552]
Would love it if I didn't have to manually refresh the various tabs for updates.

**Comment by causefx** [comment]
example please.

**Comment by goofballtech** [comment]
The one that gets me is homepage.php. The calendar won't update as things download. Since i have organizr pretty much open all the time i have to go to the homepage and refresh the frame to be able to see the colors on the calendar change as things download. 

Its not a huge deal for me personally as generally things download overnight so its just the morning refresh for new TV shows but it would be nice to have an auto-refresh on a time interval option.

**Comment by MrMxyzptlk** [comment]
Yes exactly. The dashboard only updates if I use the Organizr refresh button.

**Comment by causefx** [comment]
im trying to get calendar to auto update now, fullcalendar.io is full of bugs.

**Comment by MrMxyzptlk** [comment]
Those darn developers! Ha!

**Comment by causefx** [comment]
made progress, available soon
